tests/unit/test_charm.py

```diff
@@ -41,6 +41,7 @@ def patch_builder_init_config_from_charm(monkeypatch: pytest.MonkeyPatch):
                         arch=state.Arch.ARM64,
                         bases=(state.BaseImage.JAMMY, state.BaseImage.NOBLE),
                         juju_channels=("2.9/stable", "3.1/stable"),
+                        microk8s_channels=("",),
```

Comment:
Should there be a test where microk8s is defined?

Reply:
Added, thanks for the review!